ucl-university-college-apa.csl

```diff
@@ -1,23 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
-<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" page-range-format="expanded" default-locale="da-DK">
-  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" page-range-format="expanded">
   <info>
     <title>UCL University College - APA</title>
-    <title-short>UCL - APA</title-short>
+    <title-short>APA - UCL</title-short>
     <id>http://www.zotero.org/styles/ucl-university-college-apa</id>
     <link href="http://www.zotero.org/styles/ucl-university-college-apa" rel="self"/>
-    <link href="http://www.zotero.org/styles/apa" rel="template"/>
-    <link href="https://sites.google.com/ucl.dk/ucl-formelle-krav/apa" rel="documentation"/>
+    <link href="http://www.zotero.org/styles/apa-6th-edition" rel="template"/>
+    <link href="https://apastyle.apa.org/style-grammar-guidelines/references/examples" rel="documentation"/>
     <contributor>
-      <name>Camilla K. Andresen</name>
-      <email>ckan1@ucl.dk</email>
+      <name>Simon S. Hansen</name>
+      <email>ssha2@ucl.dk</email>
     </contributor>
```

Comment:
@ssha4900, since this looks like a modification of the existing style, we would generally leave past contributors and just add your name (if they no longer wish to be contacted about the style we can of course remove their email address).